### ISSUE TITLE
Setting bg color in nodes glitches

### ISSUE THREAD

**Issue by radare** [radare/radare2#14443]
we should use this bgcolor ffor tracing info and maybe for try/catch
```
ec graph.box red blue
ec graph.box2 red blue
```

<img width="651" alt="Screenshot 2019-06-28 at 04 03 42" src="https://user-images.githubusercontent.com/917142/60311980-c8b5af80-9959-11e9-9fe3-45a6aee33724.png">

**Comment by Jagroop-Singh** [comment]
I'm currently taking a crack at this, I'm assuming that you want the default behavior to be setting the color of the inside of the box to bgcolor, or just simply behind some of the lines such as tracing and try catch?

**Comment by trufae** [comment]
Yes, ideally setting the whole background of each basic block would be ideal. But that use to conflict with disasm colorization, so any other approach like highlighting in bg only the node title or the borders could work too and improve the situation
